### ISSUE TITLE
yarpviz is not compiled in robotology-superbuild

### ISSUE THREAD

**Issue by traversaro** [robotology/robotology-superbuild#987]
The [`yarpviz`](https://www.yarp.it/latest/yarpviz.html) tool is a useful tool, but at the moment it is not compiled if one follows the default robotology-superbuild instructions. As yarp enables it by default the `YARP_COMPILE_yarpviz` option if the graphviz library is installed and found (see https://github.com/robotology/yarp/blob/55ebea902ab1235e3b588e5d9eca9257a7ef6180/cmake/YarpFindDependencies.cmake#L563), probably a good trade-off is just to install graphviz as a dependency, without specifying additional options.

**Comment by traversaro** [comment]
Probably we were not doing this as graphviz was missing in vcpkg, but considering that we are slowly dropping vcpkg as a way of installing dependency, and graphviz is now available thanks to @Ace314159  (see https://github.com/microsoft/vcpkg/pull/20564), we probably can enable it now.

**Comment by traversaro** [comment]
Note that on conda builds, `graphviz` was already included as a transitive dependency of gazebo, but in CI builds the yarpviz was still not building with this output:
~~~
2022-01-07T02:29:48.6151965Z --  [ ]     Compile yarpviz (dependencies unsatisfied: "YARP_HAS_Graphviz;YARP_HAS_QGVCore") (YARP_COMPILE_yarpviz)
~~~

**Comment by traversaro** [comment]
At a first glance to the CI output, the issue is now solved for apt, but is still existing for conda. I did not check for vcpkg or brew due to https://github.com/robotology/robotology-superbuild/issues/842 and https://github.com/robotology/robotology-superbuild/blob/master/doc/deprecated-installation-methods.md .

**Comment by traversaro** [comment]
> At a first glance to the CI output, the issue is now solved for apt, but is still existing for conda. I did not check for vcpkg or brew due to #842 and https://github.com/robotology/robotology-superbuild/blob/master/doc/deprecated-installation-methods.md .

I forgot about this, but the conda issue should be solved by  https://github.com/robotology/ycm/pull/389 (and perhaps propagate it to https://github.com/robotology/yarp/blob/master/extern/qgv/qgv/cmake/modules/FindGraphviz.cmake ?).

**Comment by traversaro** [comment]
Fixed by https://github.com/robotology/robotology-superbuild/pull/1077 . The first release for which this will be fixed is 2022.02.1 .

**Comment by traversaro** [comment]
> (and perhaps propagate it to https://github.com/robotology/yarp/blob/master/extern/qgv/qgv/cmake/modules/FindGraphviz.cmake ?).

This was not necessary.
